modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/IdentifyingTracker.kt

```diff
@@ -29,15 +29,15 @@ abstract class IdentifyingTracker(
         }
     }
 
-    protected val anonID: String?
+    val anonID: String?
```

Comment:
I updated this to be public because I need to access this from outside of the class, but it's safe because is `val` and the variable that is modified is actually `anonymousID`. We also do the same for uuid in this classe